src/components/ObsDetails/DetailsTab.js

```diff
@@ -0,0 +1,157 @@
+import { t } from "i18next";
+import _ from "lodash";
+import type { Node } from "react";
+import React, { useState } from "react";
+import { Menu } from "react-native-paper";
+import IconMaterial from "react-native-vector-icons/MaterialIcons";
+import useIsConnected from "sharedHooks/useIsConnected";
+
+import Attribution from "./Attribution";
+import checkCamelAndSnakeCase from "./helpers/checkCamelAndSnakeCase";
+
+type Props = {
+  observation: Object
+}
+
+const headingClass = "mt-[20px] mb-[11px] text-black";
+const sectionClass = "mx-[15px] mb-[20px]";
+
+const DetailsTab = ( { observation }: Props ): Node => {
+  const isOnline = useIsConnected( );
+  const application = observation?.application?.name;
+  const [locationKebabMenuVisible, setLocationKebabMenuVisible] = useState( false );
+
+  const displayQualityGradeOption = option => {
+    const qualityGrade = observation?.quality_grade;
+    const labelClassName = ( qualityGrade === option ) ? "font-bold" : "";
+
+    return (
+      <View className="flex-col space-y-[8px]">
+        <QualityGradeStatus qualityGrade={option} />
+        <Body4 className={labelClassName}>{_.startCase( _.camelCase( t( option ) ) )}</Body4>
+      </View>
+    );
+  };
+
+  return (
+    <>
+      {observation.description && (
+        <>
+          <View className={sectionClass}>
+            <Heading4 className={headingClass}>{t( "NOTES" )}</Heading4>
+            <Body2>{observation.description}</Body2>
+          </View>
+          <Divider />
+        </>
+      )}
+
+      <View className="flex-row justify-between items-center mt-[8px] mx-[15px]">
+        <Heading4>{t( "LOCATION" )}</Heading4>
+        <KebabMenu
+          visible={locationKebabMenuVisible}
+          setVisible={setLocationKebabMenuVisible}
+        >
+          <Menu.Item
+            onPress={async ( ) => {
+
+            }}
+            title={t( "Share-location" )}
+          />
+          <Menu.Item
+            onPress={async ( ) => {
+
+            }}
+            title={t( "Copy-coordinates" )}
+          />
+        </KebabMenu>
+      </View>
+      {isOnline
+        ? (
+          <Map
+            obsLatitude={observation.latitude}
+            obsLongitude={observation.longitude}
+            mapHeight={230}
+          />
+        ) : (
+          <View className="h-16 items-center justify-center">
+            <IconMaterial
+              name="wifi-off"
+              size={30}
+              accessibilityRole="image"
+              accessibilityLabel={t( "Location-map-unavailable-without-internet" )}
+            />
+          </View>
+        )}
+      <View className={`mt-[11px] ${sectionClass}`}>
+        <ObservationLocation observation={observation} details />
+      </View>
+      <Divider />
+
+      <View className={sectionClass}>
+        <Heading4 className={headingClass}>{t( "DATE" )}</Heading4>
+        <DateDisplay
+          classNameMargin="mb-[12px]"
+          label={t( "Observed" )}
```

Comment:
This could use a more descriptive key, like `date_observed_header` or `date_observed_header_short` or something. Otherwise translators won't know that this refers to a date and that it's a header and not some piecemeal part of a sentence.